src/components/ChatCard/ChatCard.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import ReplyCard from "../ChatReplyCard/ReplyCard";
+
+export default function ChatCard({ data, handleChatReply, chatType }) {
+  return (
+    <div
+      key={data.messageId}
+      className={`flex ${
+        chatType === "message" ? "justify-end" : "justify-start"
+      } gap-2`}
+    >
+      <div className="message-container flex flex-col">
+        {data.isRepliedChat ? (
```

Comment:
As discussed offline, we can have a property called `parentId` for checking this